docs/api/api-handler/resolve-context.md

```diff
@@ -0,0 +1,98 @@
+---
+id: resolve-context
+title: ReSolve Context
```

Comment:
On this line you say "ReSolve", but on line 7 below you say "reSolve". Which one is correct?
Can we make it consistent?

Reply:
The framework's name is canonically spelled "reSolve" but here we need to use Title Case. Up to this point, I always capitalized 'R' in titles and at the start of a sentence because "ReSolve" seemes to look better on the site's navigation panels and when there is a punctuation mark just before the word:

"... this sentence ends. ReSolve etc,, etc.,"

compare to:

"... this sentence ends. reSolve etc,, etc.,"

However, if you say that "reSolve" should always start with a lowercase 'r', I will make sure to fix it here and in other occurrences.

Reply:
Since this is more a technical matter than a grammatical matter, let's keep your method as is. :)

Reply:
OK, thank you